.github/workflows/data-product-example-2-poc.yaml

```diff
@@ -3,36 +3,40 @@ name: POC - zip and upload _example_2
 
 on:
   push:
-    branches:
-      - main
+    # branches:
+    #   - main
     paths:
-      - 'data-products/**'
-      - '.github/workflows/zip-data-products.yml'
+      - "data-products/_example_2/**"
+      - ".github/workflows/data-product-example-2-poc.yaml"
 
 jobs:
-  preview:
-    name: Test
+  test:
+    uses: .github/workflows/data-product-example-2-test.yaml
+
+  upload:
+    name: Zip and upload
     runs-on: ubuntu-latest
+    needs: test
     permissions:
       id-token: write # This is required for requesting the JWT
       contents: read # This is required for actions/checkout
     defaults:
       run:
-        working-directory: data-products
+        working-directory: data-products/_example_2
     strategy:
       max-parallel: 4
       matrix:
         python-version: [3.10.10]
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0
         with:
           fetch-depth: 2
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@d27e3f3d7c64b4bbf8e4abfb9b63b83e846e0435 # v4.5.0
         with:
           python-version: ${{ matrix.python-version }}
       - name: Install Python dependencies
-        run: pip install -r _example_2/requirements.txt
+        run: pip install -r requirements.txt
       - name: Zip data product
         run: python package_data_product.py
```

Comment:
If working directory is `data-products/_example_2`, then this needs to change as this is in the `data-products/` directory. You can either change it here by adding `../package_data_product.py` or changing where the `package_data_product.py` lives in the data_products repo

Reply:
I moved it because it's specific to _example_2